packages/web/src/common/store/social/users/sagas.ts

```diff
@@ -105,14 +105,18 @@ export function* confirmFollowUser(
   accountId: ID,
   onSuccessActions?: Action[]
 ) {
-  const audiusBackendInstance = yield* getContext('audiusBackendInstance')
+  const audiusSdk = yield* getContext('audiusSdk')
+  const sdk = yield* call(audiusSdk)
   yield* put(
     confirmerActions.requestConfirmation(
       makeKindId(Kind.USERS, userId),
       function* () {
         const { blockHash, blockNumber } = yield* call(
-          audiusBackendInstance.followUser,
-          userId
+          [sdk.users, sdk.users.followUser],
+          {
+            userId: encodeHashId(accountId),
```

Comment:
In the future, please use `Id.parse` for values that are required and `OptionalId.parse` for values that are not. Those use actual Zod schemas and will help us phase out `encodeHashId`.